### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by abhi6299700** [ubports-on-vince/ubports-ci#8]
you are still alive ????? is all working now on vince

**Comment by 0312birdzhang** [comment]
Yes, I'm still alive! But, this project is dead.

**Comment by abhi6299700** [comment]
i just wanna to ask everything working properly.....

**Comment by 0312birdzhang** [comment]
> i just wanna to ask everything working properly..... and now do u have vince or not ????

Yes i have, and updated sailfishos to [5.0](https://github.com/Sailfish-On-Vince/ci/releases/tag/0.4.1) last days.

**Comment by abhi6299700** [comment]
okk i am going to install sailfish os bro ....u know how i find u ...i found u on 4pda website. thanks to google who translated russian lang. to english.
